.github/workflows/prbuild.yml

```diff
@@ -2,23 +2,18 @@ name: PR Build
 
 on:
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
-
   build:
-    name: Build and Test
+    name: Build and test
```

Comment:
Aren't we naming it as "Build and Test" in other repos?

Reply:
I changed it to this format in all the repos.